src/libraries/sendtohelix.proj

```diff
@@ -76,6 +76,11 @@
     <_RuntimeInputs Include="$(TestHostRootPath)**/*.dll" />
   </ItemGroup>
 
+  <ItemGroup>
+    <HelixProperties Condition="'$(BUILD_BUILDURI)' != ''" Include="build" Value="$(BUILD_BUILDURI)" />
```

Comment:
Maybe this one should go directly into arcade in the Helix SDK so that all helix jobs have this info.

Reply:
is there a way to add the attempt as well? or is that included in `BUILD_BUILDURI`?

Reply:
It's not inculde in the URI. Really we should have 

- Build Number
- Buidl project
- Attempt (if possible)

All of these as properties as they're all interesting query points.

Reply:
Yeah, I think we can have all those three. I would have:

- Build Number
- Build Project
- Attempt
- Build URI

Reply:
> Maybe this one should go directly into arcade in the Helix SDK so that all helix jobs have this info.

Agree; this was merged 5 days ago in https://github.com/dotnet/arcade/pull/5480 .  Adding more details that you want (like attempt) there might be a good idea because then we have uniform coverage, and I'm pretty sure you're already using SendToHelix from the Helix Arcade SDK in Runtime.

Reply:
Thanks @MattGal -- I will definitely add the attempt to that list as well, and I'll update this to only include RuntimeFlavor.